test/integration_test/common_test.go

```diff
@@ -928,6 +932,213 @@ func getTokenFromSecret(secretName, secretNamespace string) (string, error) {
 	return "", fmt.Errorf("secret does not contain key 'auth-token'")
 }
 
+func createSecret(t *testing.T, secret_name string, secret_map map[string]string) *v1.Secret {
+	secret := &v1.Secret{
+		ObjectMeta: meta_v1.ObjectMeta{
+			Name:      secret_name,
+			Namespace: "kube-system",
+		},
+		StringData: secret_map,
+	}
+	secretObj, err := core.Instance().CreateSecret(secret)
+	if !errors.IsAlreadyExists(err) {
+		require.NoError(t, err, "failed to create secret for volumes")
+	}
+	return secretObj
+}
+
+func cleanup(t *testing.T, namespace string, storageClass string) {
```

Comment:
* We can rewrite this function to take an additional param as `source` or `destination` so that it can also be used to cleanup on source.
* This will be useful during failback when source and destination roles are reversed.

Reply:
On line 965, it already calls the funcDeleteNamespace for both source and destination.